src/components/Icon/styles.js

```diff
@@ -4,4 +4,9 @@ export const StyledIcon = styled.svg`
   display: inline-block;
   vertical-align: middle;
   width: ${({ theme, width }) => width || theme.sizes.regular};
+
+  svg {
+    fill: ${({ color, theme }) =>
+      color === 'dark' ? theme.colors.dark : theme.colors.light};
+  }
```

Comment:
I don’t think you need to target the `svg`? you’re already working on it and it should inherit the `fill`, no?
btw I see you do that a lot, it’s handy but I feel like you lose some benefits of css-in-js as you start having different styles possibly affecting the same element